Explorer/Assets/DCL/AvatarRendering/AvatarShape/Rendering/TextureArray/TextureArrayHandler.cs

```diff
@@ -45,6 +45,46 @@ public class TextureArrayHandler
                 CreateHandler(new Vector2Int(defaultResolutions[i], defaultResolutions[i]));
         }
 
+
+        public TextureArrayHandler(
+            int minArraySize,
+            IReadOnlyList<TextureArrayResolutionDescriptor> textureArrayResolutionDescriptors,
+            int arrayID,
+            int textureID,
+            TextureFormat textureFormat,
+            IReadOnlyDictionary<TextureArrayKey, Texture> defaultTextures,
+            int initialCapacityForEachResolution = PoolConstants.AVATARS_COUNT)
```

Comment:
This `ctor` is not for `Avatar` domain, you should not use this constant
`int initialCapacityForEachResolution = PoolConstants.AVATARS_COUNT`